Signal/src/util/MainAppContext.h

```diff
@@ -0,0 +1,9 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
+#import <SignalServiceKit/AppContext.h>
+
+@interface MainAppContext : NSObject <AppContext>
```

Comment:
Though currently without affect, in case we add more later, can you include the NS_ASSUME_NONNULL

Reply:
Done.